.github/workflows/e2e-chrome.yml

```diff
@@ -40,38 +55,45 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        index: [0, 1, 2, 3, 4, 5, 6, 7]
+        index: [0, 1, 2, 3, 4, 5, 6, 7, 8, 9]
     with:
       test-suite-name: test-e2e-chrome-flask
       build-command: yarn build:test:flask
       test-command: yarn test:e2e:chrome:flask
       matrix-index: ${{ matrix.index }}
       matrix-total: ${{ strategy.job-total }}
 
-  test-e2e-chrome-report:
-    needs:
-      - test-e2e-chrome
-      - test-e2e-chrome-multiple-providers
-      - test-e2e-chrome-rpc
-      - test-e2e-chrome-flask
+  test-e2e-chrome-vault-decryption:
+    uses: ./.github/workflows/run-e2e.yml
+    with:
+      test-suite-name: test-e2e-chrome-vault-decryption
+      build-artifact: prep-build-dist-browserify-chrome
```

Comment:
This doesn't have a `needs: prep-build-test-browserify`

Perhaps that's okay, because if the race happens a certain way, it will just run the `build-command`